### PR TITLE
Feature/anonymization

### PR DESCRIPTION
**Pull request by ackRow**

### CODE REVIEW

File: tests/data/export/sqlite.db
Comment (Manuthor) [inline tests/data/export/sqlite.db:1]: revert this change?

File: scripts/ci_install_pyo3_builds.sh [hunk 1]
```diff
@@ -8,9 +8,9 @@ install_lib() {
         rm linux.zip && rm -rf x86_64*
 }
 
-install_lib "v1.2.1"
+install_lib "v1.4.0"
```
Comment (Manuthor) [inline scripts/ci_install_pyo3_builds.sh:11]:
```suggestion
install_lib "v2.1.0"
```
2.1.0 should be the next release version

Comment (Manuthor) [inline scripts/ci_install_pyo3_builds.sh:11]:
```suggestion
install_lib "v2.1.0"
```
2.1.0 should be the next release version

File: src/cloudproof_py/anonymization/anonymize.py [hunk 2]
```diff
@@ -0,0 +1,198 @@
...
+)
+from cloudproof_fpe import Alphabet, Integer
+from dateutil import parser as date_parser
+from humps import decamelize
+
+DURATION_IN_SECONDS = {
+    "Second": 1,
+    "Minute": 60,
+    "Hour": 3600,
+    "Day": 86400,
```
Comment (Manuthor) [inline src/cloudproof_py/anonymization/anonymize.py:25]: no week?

File: src/cloudproof_py/anonymization/anonymize.py [hunk 3]
```diff
@@ -0,0 +1,198 @@
...
+    mean: Optional[float] = None,
+    std_dev: Optional[float] = None,
+    min_bound: Optional[float] = None,
+    max_bound: Optional[float] = None,
+):
+    """
+    Returns a `NoiseGenerator` object based on the specified options.
+
+    Args:
+        method (str): The distribution to use for generating noise.
```
Comment (Manuthor) [inline src/cloudproof_py/anonymization/anonymize.py:43]: add the possible distribution names? (Laplace, Gaussian or Uniform)

File: src/cloudproof_py/anonymization/anonymize.py [hunk 4]
```diff
@@ -0,0 +1,198 @@
...
+        option_type (str): Whether the noise options are specified as parameters or bounds.
```
Comment (Manuthor) [inline src/cloudproof_py/anonymization/anonymize.py:44]: maybe add a hint

File: src/cloudproof_py/anonymization/anonymize.py (hunk 2 above)
Comment (ackRow) [reply src/cloudproof_py/anonymization/anonymize.py:25]: good question, we though about that with Celia but decided not to add it

File: tests/data/export/sqlite.db
Comment (ackRow) [reply tests/data/export/sqlite.db:1]: Since the update of Findex 4.0, we need to regenerate this file.

**Review by Manuthor — APPROVED** [review]
